_data/projects/evergreen-ils.yml

```
@@ -0,0 +1,12 @@
+name: Evergreen
+desc: Highly-scalable software for libraries that helps library patrons find library materials, and helps libraries manage, catalog, and circulate those materials.
+site: https://evergreen-ils.org/
```

@sandbergja The links that you gave for contributing are not available easily from here, for a newbie or beginner it is hard to go to this site and search how to contribute. It will be better if you give some other link here or have this site with more information.

---

This is good feedback, thanks.  How about this?
```suggestion
site: https://evergreen-ils.org/involvement/
```
I will also bring your feedback to the community to see if we can clarify the path from the main page to finding out how to contribute.

---

That will be great @sandbergja